src/main/java/racingcar/RacingGameManager.java

```diff
@@ -5,46 +5,47 @@
 public class RacingGameManager {
     private static final ArrayList<Car> cars = new ArrayList<>();
     private static int raceCount;
+
     public static void startRace() {
-        carEntry();
+        entryCar();
         runRace();
         printWinner();
     }
 
-    public static void carEntry() {
+    public static void entryCar() {
         Output.inputNameMessage();
         ArrayList<String> names = Input.inputName();
         Output.inputNumberMessage();
         raceCount = Input.inputNumber();
-        for(String name: names) {
+        for (String name : names) {
             cars.add(new Car(name));
         }
     }
 
     public static void runRace() {
         Output.resultMessage();
-        for(int i=0; i<raceCount; i++) {
+        for (int i = 0; i < raceCount; i++) {
             ArrayList<Car> cars = runRaceOneTime();
-            Output.printResult(cars);
+            Output.result(cars);
         }
     }
 
     public static ArrayList<Car> runRaceOneTime() {
-        for(Car car: cars) {
-            car.forward();
+        for (Car car : cars) {
+            car.move();
         }
         return cars;
     }
 
     public static void printWinner() {
-        Output.printWinner(whoIsWinner(cars));
+        Output.winner(findWinner(cars));
     }
 
-    public static ArrayList<String> whoIsWinner(ArrayList<Car> cars) {
+    public static ArrayList<String> findWinner(ArrayList<Car> cars) {
         ArrayList<String> winners = new ArrayList<>();
         int highScore = findHighScore(cars);
-        for(Car car: cars) {
-            if(car.getRaceScore() == highScore){
+        for (Car car : cars) {
+            if (car.getMoveDistance() == highScore) {
```

Comment:
이 부분은 차가 움직인 거리가 highScore 와 같은지 확인하는 조건문이네요!
즉, 해당 차가 winner 인지 아닌지 판단하는 역할을 하고 있어요. 
이런 경우, 단순하게 `moveDistance` 를 getter 로 가져와서 highScore 와 비교하기보다, Car 의 메서드로 넣어서 '의미를 부여해주는 것'이 좋습니다 :) 
다음과 같이 써볼 수 있을 것 같아요!

```suggestion
            if (car.isWinner(highScore)) {
```

Reply:
```
    public boolean isWinner(int highScore){
        return this.moveDistance == highScore;
    }
```

조언해 주신대로 Car에 isWinner 메서드를 추가하여 수정했습니다!